Client.xcodeproj/project.pbxproj

```diff
@@ -149,7 +149,7 @@
 		28E08C9A1AF44F00009BA2FA /* BrowserSchema.swift in Sources */ = {isa = PBXBuildFile; fileRef = 282915E51AF1A7920006EEB5 /* BrowserSchema.swift */; };
 		28E23C121AC5A5EE00F5AC85 /* State.swift in Sources */ = {isa = PBXBuildFile; fileRef = 28E23C111AC5A5EE00F5AC85 /* State.swift */; };
 		28E91E751B443AD5009DF274 /* SyncConstants.swift in Sources */ = {isa = PBXBuildFile; fileRef = 28E91E741B443AD5009DF274 /* SyncConstants.swift */; };
-		28EADE5D1AFC3A78007FB2FB /* UIImageViewExtensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 28EADE381AFC3898007FB2FB /* UIImageViewExtensions.swift */; };
+		28EADE5D1AFC3A78007FB2FB /* UIImageView+Utilies.swift in Sources */ = {isa = PBXBuildFile; fileRef = 28EADE381AFC3898007FB2FB /* UIImageView+Utilies.swift */; };
```

Comment:
@thatswinnie  UIImageView+Utilies -> UIImageView+Utils or UIImageView+Utilities as I am not sure if Utilies is a proper name 🤔

Reply:
😆 `Utils` it is then

Reply:
This is the same theme for all. If anything I'd go with `+ Extensions` or `+ Utils` or + `Utilities`

I feel + Extensions sound a work better as we are dealing with swift extensions and not obj-c categories

Reply:
Ah, saw it too late. I renamed it to `+Utils` already. `+Extensions` would be ok as well. How strongly do you feel about it?